src/AutofacSerilogIntegration/ContextualLoggingModule.cs

```diff
@@ -16,6 +16,7 @@ internal class ContextualLoggingModule : Module
         readonly ILogger _logger;
         readonly bool _autowireProperties;
         readonly bool _skipRegistration;
+        private bool _dispose;
```

Comment:
Can be `readonly`.

Nit: the codebase doesn't use explicit `private` modifiers.